frontend/packages/policy-editor/src/components/AddPolicyRuleButton/AddPolicyRuleButton.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import classes from './AddPolicyRuleButton.module.css';
+import { PlusIcon } from '@studio/icons';
+import { Paragraph } from '@digdir/design-system-react';
+import { usePolicyEditorContext } from '../../contexts/PolicyEditorContext';
+import type { PolicyRuleResource, PolicyRuleCard } from '../../types';
+import { emptyPolicyRule, createNewPolicyResource, getNewRuleId } from '../../utils';
+import { useTranslation } from 'react-i18next';
+
+export type AddPolicyRuleButtonProps = {
+  onClick: () => void;
+};
+
+export const AddPolicyRuleButton = ({ onClick }: AddPolicyRuleButtonProps): React.ReactNode => {
+  const { policyRules, setPolicyRules, savePolicy, usageType, resourceId, resourceType } =
+    usePolicyEditorContext();
+  const { t } = useTranslation();
+
+  const handleAddCardClick = () => {
+    onClick();
+
+    const newResource: PolicyRuleResource[][] = [
+      createNewPolicyResource(usageType, resourceType, resourceId),
+    ];
+    const newRuleId: string = getNewRuleId(policyRules);
+
+    const newRule: PolicyRuleCard = {
+      ...emptyPolicyRule,
+      ruleId: newRuleId,
+      resources: newResource,
+    };
+
+    const updatedRules: PolicyRuleCard[] = [...policyRules, ...[newRule]];
+
+    setPolicyRules(updatedRules);
+    savePolicy(updatedRules);
+  };
+
+  return (
+    <button className={classes.button} type='button' onClick={handleAddCardClick}>
+      <Paragraph size='small'>{t('policy_editor.card_button_text')}</Paragraph>
+      <PlusIcon className={classes.icon} />
+    </button>
+  );
```

Comment:
No, that is not possible 😄 In the design we have now, the add button looks the same as the expandable policy rule cards, shown in picture below. However, it might change in a future design 😄 
<img width="647" alt="image" src="https://github.com/Altinn/altinn-studio/assets/133344438/e9d87d0c-4e9e-4585-975e-6d9a6a9d3fc8">

Reply:
haha ok :D